src/components/Banners/Banner.tsx

```diff
@@ -0,0 +1,74 @@
+import React from "react";
+import styled from "@emotion/styled";
+import { uiColors } from "@leafygreen-ui/palette";
+import Icon from "@leafygreen-ui/icon";
+import { Body } from "@leafygreen-ui/typography";
+import { BannerTypeKeys } from "types/banner";
+
+const { green, red, yellow, blue, gray } = uiColors;
+
+interface Props {
+  id?: string;
+  type: BannerTypeKeys;
+  removeBanner?: (bannerId: string) => void;
+}
+
+export const Banner: React.FC<Props> = ({
+  children,
+  type,
+  removeBanner,
+  id,
+}) => {
+  const onClickX = () => removeBanner(id);
+  return (
+    <Wrapper type={type}>
+      <StyledBody type={type}>{children}</StyledBody>
+      {(type === "success" || type === "info") && (
```

Comment:
why not allow warning banners to be removable?

Reply:
because designs